### PR TITLE
EN-5528 : fix bug with bootstrap from storage infinite loop

### PR DESCRIPTION
**Pull request by miiu96**

### CODE REVIEW

File: core/statistics/softwareVersion/softwareVersion.go [hunk 1]
```diff
@@ -18,6 +19,7 @@ type tagVersion struct {
 	TagVersion string `json:"tag_name"`
 }
 
+// SoftwareVersionChecker is component used to check if a new software stable tag in available
```
Comment (bogdan-rosianu) [inline core/statistics/softwareVersion/softwareVersion.go]: // SoftwareVersionChecker is a component which is used to check if a new software stable tag is available

File: core/statistics/softwareVersion/softwareVersion.go [hunk 2]
```diff
@@ -32,8 +34,14 @@ func NewSoftwareVersionChecker(appStatusHandler core.AppStatusHandler) (*Softwar
 		return nil, core.ErrNilAppStatusHandler
 	}
 
-	// check interval will be random in a interval [1hour, 1hour 15minutes]
-	randInterval := time.Duration(rand.Int() % 15)
+	// check interval will be random in a interval [1hour5minutes , 1hour20minutes]
```
Comment (bogdan-rosianu) [inline core/statistics/softwareVersion/softwareVersion.go]: `// check interval will be a random duration in the interval [1hour5minutes , 1hour20minutes]`

File: process/sync/storageBootstrap/baseStorageBootstrapper.go [hunk 3]
```diff
@@ -60,6 +60,12 @@ func (st *storageBootstrapper) loadBlocks() error {
 		if err != nil {
 			break
 		}
+
+		if round == headerInfo.LastRound {
+			err = sync.ErrCorruptBootstrapFromStorageDb
```
Comment (sasurobert) [inline process/sync/storageBootstrap/baseStorageBootstrapper.go]: errors are shadowed by line 100, add log.Warn for each error.

File: core/statistics/softwareVersion/softwareVersion.go (hunk 1 above)
Comment (miiu96) [reply core/statistics/softwareVersion/softwareVersion.go]: done.

File: core/statistics/softwareVersion/softwareVersion.go (hunk 2 above)
Comment (miiu96) [reply core/statistics/softwareVersion/softwareVersion.go]: done.

File: process/sync/storageBootstrap/baseStorageBootstrapper.go (hunk 3 above)
Comment (miiu96) [reply process/sync/storageBootstrap/baseStorageBootstrapper.go]: done.

**Pull request merged**